### PR TITLE
[Rule Tuning] Okta User Sessions Started from Different Geolocations

### PR DESCRIPTION
**Pull request by terrancedejesus**
## Issues
* https://github.com/elastic/ia-trade-team/issues/271
* https://github.com/elastic/detection-rules/issues/3438

## Summary
This pull request tunes the `Okta User Sessions Started from Different Geolocations` after recommendation from the related issue. Note, we have added an investigation guide as well tailored for this activity. In addition, we have converted this to ES|QL.

Screenshot 2024-06-18 at 11 03 16 AM

### CODE REVIEW

**Review by Aegrah — APPROVED** [review]

File: rules/integrations/okta/initial_access_okta_user_sessions_started_from_different_geolocations.toml [hunk 1]
```diff
@@ -4,63 +4,104 @@ integration = ["okta"]
...
-note = """## Setup
+note = """
+
+## Triage and analysis
+
+### Investigating Okta User Sessions Started from Different Geolocations
+
+This rule detects when a specific Okta actor has multiple sessions started from different geolocations. Adversaries may attempt to launch an attack by using a list of known usernames and passwords to gain unauthorized access to user accounts from different locations.
 
-The Okta Fleet integration, Filebeat module, or similarly structured data is required to be compatible with this rule."""
```
Comment (shashank-elastic) [inline rules/integrations/okta/initial_access_okta_user_sessions_started_from_different_geolocations.toml:23]: Nit: Can we restore the Setup?

**Review by shashank-elastic — APPROVED** [review]
Nit: Apart from having a Setup information the rule looks good. 
We have added great informations of Triage and Analysis, it would be good to atleast restore the skeletal Setup information for the integration.

File: rules/integrations/okta/initial_access_okta_user_sessions_started_from_different_geolocations.toml [hunk 2]
```diff
@@ -4,63 +4,104 @@ integration = ["okta"]
 maturity = "production"
 min_stack_comments = "Breaking change in Okta integration bumping version to ^2.0.0"
 min_stack_version = "8.10.0"
```
Comment (shashank-elastic) [inline rules/integrations/okta/initial_access_okta_user_sessions_started_from_different_geolocations.toml:6]: https://github.com/elastic/detection-rules/pull/3797#discussion_r1647202783

Comment (shashank-elastic) [reply rules/integrations/okta/initial_access_okta_user_sessions_started_from_different_geolocations.toml:6]: Referencing the min stack review from a cross PR as i observed this is also an ESQL change.

Comment (terrancedejesus) [reply rules/integrations/okta/initial_access_okta_user_sessions_started_from_different_geolocations.toml:6]: Updated, thanks for pointing that out!

File: rules/integrations/okta/initial_access_okta_user_sessions_started_from_different_geolocations.toml (hunk 1 above)
Comment (terrancedejesus) [reply rules/integrations/okta/initial_access_okta_user_sessions_started_from_different_geolocations.toml:23]: Restored!

**Pull request merged**